modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/BindConfigurationMVCActionCommand.java

```diff
@@ -128,7 +133,47 @@ public boolean processAction(
 			properties.put(ConfigurationAdmin.SERVICE_FACTORYPID, factoryPid);
 		}
 
-		configureTargetService(configurationModel, configuration, properties);
+		try {
+			configureTargetService(
+				configurationModel, configuration, properties);
+		}
+		catch (Exception e) {
```

Comment:
Let's never catch Exception. You will eventually catch an exception you shouldn't. Also, once you add this line, it's super hard to turn it into a more specific catch safely.

Catch the more specific exception that makes sense. In this case ConfigurationModelActionException (or after my proposed rename ConfigurationModelListenerException)